include/unistd.h

```diff
@@ -44,6 +44,8 @@
 #include <sys/types.h>
 #include <nuttx/compiler.h>
 
+#include <getopt.h>
```

Comment:
Should not include this here. Non-standard macros and identifiers in getopt.h can break conforming POSIX programs, for example valid declaration:  `bool no_argument = true;`

Reply:
Done.